### PR TITLE
docstings in game.board

### PR DESCRIPTION
**Pull request by Average-user**

### CODE REVIEW

File: src/cljc/game_of_ur/game/board.cljc [hunk 1]
```diff
@@ -124,14 +153,19 @@
       (cond (= 0 roll) (assoc board :turn next-turn)
             (valid-move? board full-move) (assoc next-board :turn next-turn)))))
 
-(defn all-moves [{:keys [turn] :as board} roll]
+(defn all-moves
+  "returns all posible moves with a given roll assuming
+   an empty board"
```
Comment (polymeris) [inline src/cljc/game_of_ur/game/board.cljc]: Not really an empty board. It just generates moves originating from every cell on the board.

File: src/cljc/game_of_ur/game/board.cljc [hunk 2]
```diff
@@ -64,44 +64,69 @@
...
+  (let [in-destination (get stones destination)]
+    (and destination
+         (= turn player)                          
+         (or (not= origin :home)    (> (home player) 0))
+         (or (= origin :home)       (= player (get stones origin)))
+         (or (nil? in-destination)  (= (opponent turn) in-destination))
+         (or (nil? in-destination)  (not (rosettes destination))))))
+
+(defn move-stone
+  "- If the origin is in the board, it replace the content of origin with nil
```
Comment (polymeris) [inline src/cljc/game_of_ur/game/board.cljc]: typo

File: src/cljc/game_of_ur/game/board.cljc (hunk 1 above)
Comment (Average-user) [reply src/cljc/game_of_ur/game/board.cljc]: "returns all possible moves with a given roll, not taking in count if they are valid or not"  would be right?

Comment (polymeris) [reply src/cljc/game_of_ur/game/board.cljc]: sounds good to me

**Pull request merged**